pkgs/servers/samba/4.x.nix

```diff
@@ -19,11 +19,11 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "samba-${version}";
-  version = "4.4.6";
+  version = "4.4.8";
```

Comment:
Do we still want this given that https://github.com/NixOS/nixpkgs/commit/b0a1028a1a18932aced53f36f0a80a3a213f91e4 is in staging?